addons/knowledge/static/src/components/permission_panel/permission_panel.js

```diff
@@ -134,7 +134,7 @@ class PermissionPanel extends Component {
                     partner_id: member.based_on ? member.partner_id: false,
                 }
             }).then(res => {
-                if (self._onChangedPermission(res, willLoseAccess||willLoseWrite, willLoseWrite ? this.props.article_id : false)) {
+                if (self._onChangedPermission(res, willLoseAccess||willLoseWrite, willLoseWrite ? false : this.props.article_id)) {
```

Comment:
is this a bug fix ?

Reply:
Indeed. If the user loses access, the page should be reloaded on a different article (so this value should be false), and otherwise it should stay on the same article. Should I put it in a separate commit?
Side note: since latest changes, this doesn't seem to work correctly. To check.